packages/chrome-extension/README.md

```diff
@@ -0,0 +1,101 @@
+<p align="center">
+  <a href="https://clerk.com?utm_source=github&utm_medium=clerk_chrome_extension" target="_blank" rel="noopener noreferrer">
+    <img src="https://images.clerk.com/static/logo-light-mode-400x400.png" height="64">
+  </a>
+  <br /> 
+</p>
+
+# @clerk/chrome-extension
+
+<div align="center">
+
+[![Chat on Discord](https://img.shields.io/discord/856971667393609759.svg?logo=discord)](https://discord.com/invite/b5rXHjAg7A)
+[![Clerk documentation](https://img.shields.io/badge/documentation-clerk-green.svg)](https://clerk.com/docs?utm_source=github&utm_medium=clerk_chrome_extension)
+[![Follow on Twitter](https://img.shields.io/twitter/follow/ClerkDev?style=social)](https://twitter.com/intent/follow?screen_name=ClerkDev)
+
+[Changelog](https://github.com/clerkinc/javascript/blob/main/packages/chrome-extension/CHANGELOG.md)
+·
+[Report a Bug](https://github.com/clerkinc/javascript/issues/new?assignees=&labels=bug&template=bug_report.md&title=Bug%3A+)
+·
+[Request a Feature](https://github.com/clerkinc/javascript/issues/new?assignees=&labels=enhancement&template=feature_request.md&title=Feature%3A+)
+·
+[Ask a Question](https://github.com/clerkinc/javascript/issues/new?assignees=&labels=question&template=ask_a_question.md&title=Support%3A+)
+
+</div>
+
+---
+
+## Overview
+
+[Clerk](https://clerk.com?utm_source=github&utm_medium=clerk_chrome_extension) is the easiest way to add authentication and user management to your chrome extension. To gain a better understanding of the Clerk React SDK and [Frontend API](https://reference.clerk.com/reference/frontend-api-reference), refer to
+the <a href="https://clerk.com/docs/reference/node/getting-started?utm_source=github&utm_medium=clerk_chrome_extension" target="_blank">Node SDK</a> and <a href="https://clerk.com/docs/reference/backend-api" target="_blank">Backend API</a> documentation.
+
+## Getting Started
+
+To use this package you should first create a Clerk application and retrieve a `Publishable Key` for you application to be used as environment variables `REACT_APP_CLERK_PUBLISHABLE_KEY`.
+
+### Prerequisites
+
+- Node.js v14+
+
+### Installation
+
+```shell
+npm install @clerk/chrome-extension
+```
+
+## Usage
+
+TODO: use scenarios from ticket description
```

Comment:
We can add a link to the starter repo and also add a snippet of the <ClerkProvider/>

Reply:
Could you take a look at this: https://github.com/clerkinc/javascript/pull/971/commits/889d8d3ef342cc92b43352952e453ebbec9d104e